dragster.js

```diff
@@ -201,18 +202,20 @@
          * @return {Array}
          */
         wrapDraggableElements = function (elements) {
-            elements.forEach(function (draggableElement) {
-                var wrapper = createElementWrapper(),
-                    draggableParent = draggableElement.parentNode;
+            if (finalParams.wrapDraggableElements === TRUE) {
```

Comment:
I would do it a different way.

```
wrapDraggableElements = function (elements) {
    if (finalParams.wrapDraggableElements === FALSE) {
        return FALSE;
    }
    
    elements.forEach(function (draggableElement) {
        var wrapper = createElementWrapper(),
            draggableParent = draggableElement.parentNode;

        if (draggableParent.classList.contains(CLASS_DRAGGABLE)) {
            return FALSE;
        }

        draggableParent.insertBefore(wrapper, draggableElement);
        draggableParent.removeChild(draggableElement);
        wrapper.appendChild(draggableElement);
    });
};
```
It avoids the indentation of huge block of code.